features/ownership/approve.feature

```diff
@@ -9,6 +9,7 @@ Feature: Test approving ownership
     Given I create a minimal organizer and save the "id" as "organizerId"
     And I am authorized as JWT provider v2 user "invoerder_ownerships"
     And I request ownership for "auth0|64089494e980aedd96740212" on the organizer with organizerId "%{organizerId}" and save the "id" as "ownershipId"
+    And I wait 4 seconds
```

Comment:
As discussed in a Huddle, I think the wait needs to happen after the approval. Because in this scenario, there are 2 mails. The first mail from the request and the second mail from the approval.

Reply:
I think the easiest is just to put the wait before the `"ownership-approved" mail has been sent from`